callisto/delivery/models.py

```diff
@@ -12,36 +10,40 @@
 from django.db import models
 from django.template import Context, Template
 from django.utils import timezone
-from django.utils.crypto import get_random_string, pbkdf2
+from django.utils.crypto import get_random_string
 from django.utils.html import strip_tags
 
+from callisto.delivery.hashers import get_hasher, identify_hasher
+
+# change this to the value previously used
+ORIGINAL_KEY_ITERATIONS = 100
```

Comment:
Can you set this to 100000 (current value for the instances in prod)? it'll make any locally stored records undecryptable, but I think that's an ok tradeoff rather than trying to read it from the existing setting and potentially having to keep that around forever.